model/registry.go

```diff
@@ -14,110 +14,46 @@
 
 package model
 
-import (
-	"fmt"
-	"regexp"
-
-	"github.com/golang/protobuf/proto"
-)
+import "github.com/golang/protobuf/proto"
 
 // ConfigKey is the identity of the configuration object
 type ConfigKey struct {
 	// Config object kind, e.g.  "MyKind"
 	Kind string
 	// Config object name, e.g. "my-name"
 	Name string
-	// Config version, e.g. "prod", "canary", "v1".
-	// "default" version is reserved, use ""
-	Version string
+	// Config object namespace, e.g. "default"
+	Namespace string
 }
 
 // Config object holds the normalized config objects defined by Kind schema
 type Config struct {
 	ConfigKey
-	// Content holds the configuration struct
-	Content interface{}
+	// Spec holds the configuration struct
+	Spec interface{}
+	// Status holds the status information (may be null)
```

Comment:
what sort of Status information do expect to be here?  Perhaps we could add some examples?  interface{} makes it hard to get a feel.